### PR TITLE
Fix broken relative links & a few other typos

### PR DESCRIPTION
**Pull request by rstein66**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,63 +69,63 @@ API
...
     Ending:
     - `json`
     - `html`
     - `svg` ![](https://img.shields.io/badge/First%20Timers-1-blue.svg)
-  - `POST /organization<ENDING>`  
+  - `POST /organization<ENDING>`
     Submit an `organization` for scraping.
     This shows an html page with a link to the status of the organization.
 - Repository
-  - `GET /repositories<ENDING>`
```
Comment (niccokunzmann) [inline README.md:108]: Two spaces at the end are a newline in markdown. Would you like to remove the newline?

**Comment by niccokunzmann** [comment]
Thanks! Do you intend to remove the newline?

**Comment by niccokunzmann** [comment]
It looks ok to me :)

**Comment by rstein66** [comment]
Oh! I accidentally enabled remove trailing whitespace for markdown... gald to realize that sooner rather than later! Thanks :)

**Pull request merged**